Library/Homebrew/tap.rb

```diff
@@ -82,6 +82,16 @@ def git?
     (path/".git").exist?
   end
 
+  # The issues URL of this {Tap}.
+  # e.g. `https://github.com/user/homebrew-repo/issues`
+  def issues_url
+    if official?
+      "https://github.com/Homebrew/homebrew-#{repo.strip_prefix("homebrew-")}/issues"
+    else
+      "https://github.com/#{name}/issues"
```

Comment:
Yeah I started a long `case remote` for the most common remote patterns (e.g. GitHub/Bitbucket/etc) but it’s endless; I don’t know what we should show for custom remotes.

Also my tap remotes are like `gh:Homebrew/homebrew-php.git` where `gh` is an SSH shortcut for `git@github.com`. This means that e.g. `Tap.fetch("homebrew", "homebrew-emacs").custom_remote?` is true but we’re still able to display the issues URL because we know it’s an official tap.